src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -347,6 +410,19 @@ subroutine propagate_int_tide(h, tv, Nb, Rho_bot, dt, G, GV, US, inttide_input_C
                    ! It can be 1 point smaller if teleport is not used.
   endif
 
+  call pass_var(cn,G%Domain)
+
+  if (CS%debug) then
+    call hchksum(cn(:,:,1), "CN mode 1", G%HI, haloshift=0, scale=US%L_to_m*US%s_to_T)
+    call hchksum(CS%w_struct(:,:,:,1), "Wstruct mode 1", G%HI, haloshift=0)
+    call hchksum(CS%u_struct(:,:,:,1), "Ustruct mode 1", G%HI, haloshift=0, scale=US%m_to_Z)
+    call hchksum(CS%u_struct_bot(:,:,1), "Ustruct_bot mode 1", G%HI, haloshift=0, scale=US%m_to_Z)
+    call hchksum(CS%u_struct_max(:,:,1), "Ustruct_max mode 1", G%HI, haloshift=0, scale=US%m_to_Z)
+    call hchksum(CS%int_w2(:,:,1),   "int_w2", G%HI, haloshift=0, scale=GV%H_to_m)
```

Comment:
In these checksums, instead of scaling with `GV%H_to_m`, which depends on the Boussinesq reference density in non-Boussinesq mode, consider scaling by `GV%H_to_mks`, which does not.

Reply:
This comment was also intended to apply to the scaling factors for `CS%int_U2` and `CS%int_N2w2`, which should have scaling factors of `scale=GV%H_to_mks*US%m_to_Z**2` and `scale=GV%H_to_mks*US%s_to_T**2`.